runtime/extensions/evm/src/executor/wrapper.rs

```diff
@@ -532,6 +542,26 @@ impl<'config, S: StackStateT<'config>> TraceExecutorWrapper<'config, S> {
 				}
 			}
 		}
+		if let Some(precompile) = self.precompile {
+			if let Some(ret) = (precompile)(address, &data, Some(gas_limit), &context) {
+				return match ret {
+					Ok((s, out, cost)) => {
+						let _ = self
```

Comment:
@notlesh isn't this also what you recently fixed ?

Reply:
It is, however in our wrapper we need to do exactly what the evm is doing, even if it's wrong! Otherwise the trace won't be replicating the original transaction behaviour.

Of course once [the PR](https://github.com/rust-blockchain/evm/pull/24) is merged we can include that change in our wrapper. Once again, this can be avoided with the hook.

Reply:
Yes, it looks like it should effectively do the same thing.